src/views/SystemIntake/Documents/UploadForm.tsx

```diff
@@ -221,6 +229,42 @@ const UploadForm = () => {
             )
           }
 
+          {/* Document type */}
+          <Controller
+            name="version"
+            control={control}
+            // eslint-disable-next-line @typescript-eslint/no-shadow
+            render={({ field, fieldState: { error } }) => (
+              <FormGroup error={!!error}>
+                <Fieldset
+                  legend={
+                    <span className="text-bold">
+                      {t('intake:documents.versionLabel')}
+                    </span>
+                  }
+                >
+                  {error && (
+                    <ErrorMessage>
+                      {t('technicalAssistance:errors.makeSelection')}
+                    </ErrorMessage>
+                  )}
+                  {['CURRENT', 'HISTORICAL'].map(val => (
```

Comment:
Same as here for the enum comments -- seems like we could probably use a generated type (like `src/gql/gen/graphql`'s `SystemIntakeDocumentVersion`!

Reply:
This can be achieved with Object.values(SystemIntakeDocumentVersion).map, but it alphabetizes the options. This happens to work out with Current/Historical occuring alphabetically, but I'm unsure if it's worth mixing and matching strategies in the component.